atest/interpreter.py

```diff
@@ -39,7 +39,7 @@ def _get_interpreter(self, path):
 
     def _get_name_and_version(self):
         try:
-            output = subprocess.check_output(self.interpreter + ['-V'],
+            output = subprocess.check_output([self.interpreter[0]] + ['-V'],
```

Comment:
This doesn't look right. I'm fairly certain it breaks running tests on Windows like `atest/run.py "py -2" atest/robot/...`.

Reply:
Might be. What would be the best way to solve that?

I can only think about looking for a “python -m” or “coverage” pattern with self.interpreter and use self.interpreter[0] if above present. 

BTW, what is “py -2”?

Reply:
I don't have any immediate good ideas how to handle this. My priority at the moment is getting RF 3.2 rc 1 ready, but I can think more about this one after that. Might be better to use Slack for further discussion, though.

`py -2` is how you run Python 2 using the [Python launcher for Windows](https://docs.python.org/3/using/windows.html#python-launcher-for-windows).

Reply:
This now addressed via `--coverage` option.